cmake/SetupCodeChecks.cmake

```diff
@@ -206,8 +217,121 @@ macro(blt_add_code_checks)
                                  SRC_FILES         ${_c_sources})
     endif()
 
+    if (CLANGQUERY_FOUND)
+        set(_clang_query_target_name ${arg_PREFIX}_clang_query_check)
+        blt_error_if_target_exists(${_clang_query_target_name} ${_error_msg})
+        blt_add_clang_query_target( NAME           ${_clang_query_target_name}
+                                 WORKING_DIRECTORY ${CMAKE_BINARY_DIR}
+                                 SRC_FILES         ${_c_sources})
+    endif()
+
 endmacro(blt_add_code_checks)
 
+##-----------------------------------------------------------------------------
+## blt_configure_clang_query( CLANG_QUERY_DIR
+##                          )
+##
+## Configures the clang query helper script and finds clang query
+##
+##-----------------------------------------------------------------------------
+macro(blt_configure_clang_query)
```

Comment:
Are you expecting users to call this?
If not, we have a ``BLTPrivateMacros.cmake`` file...

Reply:
This macro doesn't exist anymore